scenariorun/src/main/kotlin/com/cosmotech/scenariorun/ContainerFactory.kt

```diff
@@ -394,6 +403,28 @@ class ContainerFactory(
             ))
   }
 
+  internal fun checkContainerImages(containers: List<ScenarioRunContainer>) {
+    // This method only checks public container registry github
```

Comment:
In my opinion, seems to be useless and time consuming to do this each time a run is launched.
It could be replaced by health endpoint, documentation, process or something else